swift_browser_ui_frontend/src/common/lang.js

```diff
@@ -439,6 +445,8 @@ let default_translations = {
       objects: {
         objectName: "Objekti",
         editObject: "Muokataan objekti: ",
+        norename: "Objekteja ei voi nimetä uudelleen. " +
```

Comment:
Probably best to move from `Objekteja` to `Objektia`. The following line is not in plural, and having it in singular would match the Finnish translation for Bucket edits.